R/install-github.r

```diff
@@ -110,6 +110,10 @@ remote_download.github_remote <- function(x, quiet = FALSE) {
     warning("GitHub repo contains submodules, may not function as expected!",
       call. = FALSE)
 
+  if (!quiet) {
```

Comment:
Actually, looking at the codes again, it might be better to keep this as a separate message block.
Between the first message and the new message, enough number of things happen, 1) dest tempfile created, 2) src URL built, 3) authenticated, 4) checking submodules. 
So it would be good to have the two messages separate.
Let me know what you think.